frontend/src/actions/cuisines.actions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+
+export const API_URL_GET_CUISINES = 'cuisines';
+export const REQUEST_CUISINES = 'REQUEST_CUISINES';
+export const RETRIEVED_CUISINES = 'RETRIEVED_CUISINES';
+
+export const requestCuisines = () => ({ type: REQUEST_CUISINES });
```

Comment:
I highly recommend [`redux-actions`](https://redux-actions.js.org).

Reply:
looks interesting, bit simpler than the classic way of redux, definitely reviewing it